.upstream-tests/test/host_only/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+add_custom_target(libcudacxx_tu_tests)
+
+set(cpp_std_versions 11 14 17 20)
+
+set(cpp_11_exclusions)
```

Comment:
As an example: `set(cpp_11_exclusions "cuda/std/mdspan")` will get excluded from C++11 builds where it is currently asserting on unsupported dialect.